s3prl/upstream/byol_a/README.md

```diff
@@ -0,0 +1,61 @@
+# BYOL-A upstream model for SUPERB
+
+This is the upstream model of BYOL-A.
+
+**Note** that you will need to compute the statistics (i.e., mean and standard deviation) of the downstream task before training and testing on it, because BYOL-A requires pre-computed normalization statistics for each downstream task.
+
+## Usage
+
+Install nnAudio if not.
+
+    pip install nnAudio
+
+### 0. Downloading weights
+
+Download the weight from the links below.
+
+- 2048-d feature (recommended): https://github.com/nttcslab/byol-a/raw/master/pretrained_weights/AudioNTT2020-BYOLA-64x96d2048.pth
+- 1024-d feature: https://github.com/nttcslab/byol-a/raw/master/pretrained_weights/AudioNTT2020-BYOLA-64x96d1024.pth
+- 512-d feature: https://github.com/nttcslab/byol-a/raw/master/pretrained_weights/AudioNTT2020-BYOLA-64x96d512.pth
+
+### 1. Configuring the number of feature dimensions
+
+Edit your `config.yaml` to match your weight.
+
+    # Dimensions of feature representations.
+    feature_d: 2048
+
```

Comment:
Is it possible to hide this step from simply inferring this information from checkpoint?
We may consider to store the config into the checkpoint?

Reply:
Yes, I will try.